framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -794,7 +794,8 @@ def set_simulation_length(self, **data):
         is_simulator_load = common.get_from_session(common.KEY_IS_SIMULATOR_LOAD) or False
         session_burst_config = common.get_from_session(common.KEY_BURST_CONFIG)
 
-        next_form = SimulatorFinalFragment()
+        simulation_number = self.burst_service2.get_available_bursts(common.get_current_project().id).__len__() + 1
```

Comment:
I know it was not generated at this change, but pls rename `self.burst_service2` into `self.burst_service`